inc/cloner/class-cloner.php

```diff
@@ -1107,7 +1107,6 @@ public function removeDefaultBookContent( $contents ) {
 				'main-body',
 				'chapter-1',
 				'appendix',
-				'contributors',
```

Comment:
No longer needed, since contributors back matter isn't going to be default book content